<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecvTimestampWidget</class>
 <widget class="QWidget" name="RecvTimestampWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,10">
   <item>
    <layout class="QVBoxLayout" name="controlLayout">
     <item>
      <widget class="QWidget" name="controlWidget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>350</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Протокол</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="protocolComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ipLayout">
          <item>
           <widget class="QLabel" name="ipLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Ip-адрес и порт:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ipInputLayout">
            <item>
             <widget class="QLineEdit" name="ipEdit"/>
            </item>
            <item>
             <widget class="QSpinBox" name="portSpinBox">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="packetsCountLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Число пакетов:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Измерение:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Файл:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="text">
             <string>/dev/urandom</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Размер данных:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Точность:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Zero-Copy</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="resultLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
